config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,25 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
+apiVersion: cert-manager.io/v1
```

Comment:
Are we planning to use cert-manager or OLM?

Reply:
We can specify a dependency on cert-manager operator in the OLM manifest.

Reply:
The reason I'm asking is OLM serves these certs. There's no reason to offload it to cert-manager.

All you need to do is include ValidatingWebhookConfiguration in kustomize. operator-sdk/OLM will pick it up. Here's an [example](https://github.com/josefkarasek/projectrequestlimits/blob/main/bundle/manifests/projectrequestlimits.clusterserviceversion.yaml#L99) (generated by operator-sdk)

Reply:
Cool, didn't know OLM can do that. Will modify the manifests.